app/src/main/java/com/example/bigowlapp/fragments/NotificationListFragment.java

```diff
@@ -50,12 +47,12 @@ public void onStart() {
         super.onStart();
 
         RepositoryFacade repositoryFacade = RepositoryFacade.getInstance();
-
-        notificationListData = repositoryFacade.getNotificationRepository().getListOfDocumentByAttribute("receiverUid", repositoryFacade.getAuthRepository().getCurrentUser().getUid(), Notification.class);
+        notificationListData = repositoryFacade.getCurrentUserNotificationRepository()
+                .getAllDocumentsFromCollection(Notification.class);
```

Comment:
Exactly what we need to avoid cramped database calls, I think this refactoring by extraction.